src/main/java/org/jabref/gui/collab/EntryDeleteChangeViewModel.java

```diff
@@ -37,6 +35,7 @@ public void makeChange(BibDatabaseContext database, NamedCompound undoEdit) {
     @Override
     public Node description() {
         PreviewViewer previewViewer = new PreviewViewer(new BibDatabaseContext(), JabRefGUI.getMainFrame().getDialogService(), Globals.stateManager, Globals.getThemeManager());
+        previewViewer.setLayout(Globals.prefs.getPreviewPreferences().getSelectedPreviewLayout());
```

Comment:
You really should pass these all as constructor args, see  EntryAddChangeViewModel